### DISCUSSION TITLE
Feature Request: Send message to contact endpoint if an alert has been silenced

### DISCUSSION THREAD

**Discussion by sdaus** [grafana/grafana#65350] [Alerting]
Currently, when an alert is silenced, the notification policy does not send any message to the contact point. This can be confusing or misleading for some users who may not be aware of the silence. It would be great if a message could be sent if the alert has been silenced. This message could include information such as the reason and duration of the silence.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82765. Feel free to continue the discussion around this there. Thank you!
